modules/dynamic-provisioning-change-default-class.adoc

```diff
@@ -38,8 +36,7 @@ storage class:
 $ oc patch storageclass gp2 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'
 ----
 
-. Make another storage class the default by adding or modifying the
-annotation as `storageclass.kubernetes.io/is-default-class=true`.
+. Make another storage class the default by adding or modifying the annotation as `storageclass.kubernetes.io/is-default-class=true`.
```

Comment:
Suggestion: 

Make another storage class the default by adding or modifying the `storageclass.kubernetes.io/is-default-class` annotation as `true`:

Reply:
I think we can replace `adding or modifying` to `setting` as the previous words are not sounding right to me . Set is more apt , what do you think @stevsmit

Reply:
revised >>Make another storage class the default by setting the storageclass.kubernetes.io/is-default-class annotation to true:

Reply:
Sounds good to me :)